### DISCUSSION TITLE
SOLD: 4x HD200 with remotes, 2x HD-PVR, 1x USB U-IRT

### DISCUSSION THREAD

**Discussion by VortexJD** [OpenSageTV/Community#118] [Marketplace]
Cleaning out my basement and came across my old SageTV gear which hasn't been used in a while.  I'd rather give this to someone who can use it vs throwing it away.  I'd prefer to give it all together vs splitting it up.  Just looking to cover shipping costs from Ohio, US.

4x HD-200 extenders with remotes.  I think I have 5 remotes, but one or two were heavily used and some buttons were getting squishy.  I don't have a server to connect them to anymore, but I can plug them in and confirm they still boot at least.

2x HD-PVR these work for the most part, but the picture quality was never 100%.  This could be due to bad power supplies which is a known issue with these devices.  I think both original power supplies failed, and I only have 1 substitute supply to offer (it's a D-link PS with the same specs as the original, used it for years that way.

1x USB U-IRT  Used this to control 2x cable boxes hooked up to the HD-PVRs.  Still in working condition as far as I'm aware.

UPDATE: Everything has been spoken for and shipped out.

**Comment by Sagetvmaster** [comment]
If nobody's spoken up yet, I'd love to be the caretaker of your old Sage gear.  What's the best way for us to get in contact?  Shipping is fine - you can use my zipcode for an estimate :)
Sagetvmaster

**Comment by Sagetvmaster** [comment]
> Cleaning out my basement and came across my old SageTV gear which hasn't been used in a while. I'd rather give this to someone who can use it vs throwing it away. I'd prefer to give it all together vs splitting it up. Just looking to cover shipping costs from Ohio, US.
> 
> 4x HD-200 extenders with remotes. I think I have 5 remotes, but one or two were heavily used and some buttons were getting squishy. I don't have a server to connect them to anymore, but I can plug them in and confirm they still boot at least.
> 
> 2x HD-PVR these work for the most part, but the picture quality was never 100%. This could be due to bad power supplies which is a known issue with these devices. I think both original power supplies failed, and I only have 1 substitute supply to offer (it's a D-link PS with the same specs as the original, used it for years that way.
> 
> 1x USB U-IRT Used this to control 2x cable boxes hooked up to the HD-PVRs. Still in working condition as far as I'm aware.

PS - not sure what the best way to communicate is here on Github, any ideas?

**Reply by VortexJD** [reply]
Just noting here that I've reached out and working on getting this stuff to sagetvmaster.

**Comment by LoisGNS** [comment]
I'm interested.

**Reply by VortexJD** [reply]
Sorry already working with sagetvmaster.

**Reply by VortexJD** [reply]
LoisGNS, I still have 2 HD200s and 1 HD-PVR remaining.  Are you still interested in any of these?

**Comment by LoisGNS** [comment]
I'm definitely interested in the HD 200s. Remind me of the functionality of the hdpvr with Sage? I assume the remotes are already spoken for.

**Reply by VortexJD** [reply]
HDPVR is a device from hauppauge to record a component/composite signal via USB connection to the PC.  I used it to record from my cable company's cable set top box.  It was a bit clunky, but it got the job done back in the day.  I still have remotes for the HD200s, but be warned they are used and they are notorious for buttons getting squishy and not working well.  I don't have the setup needed to really test how well these are working, but they worked enough for me to go into settings and confirm the HD200 was getting an IP address.

**Reply by LoisGNS** [reply]
Ok, thanks. I'd like the HD 200s and the remotes (some of mine are flaky, so always looking for dupes even if also flaky). Given the way USPS works, shipping may be the same, or not much more if you add the HDPVR. If so, I'll take that too and see what I might be able to do with it (unless there's someone else really keen on the HDPVR). I'm in central NJ, so let me know shipping costs. PayPal OK?

**Reply by VortexJD** [reply]
We can exchange private info here (might have to accept the invite first in your email)

https://github.com/VortexJD/stuff2/issues/1#issue-2177814579

**Reply by ZadkielOne** [reply]
I have at least 5 HD300's, 1 HD200, 3 Hauppauge video recorders, probably some other misc stuff including the a few original boxes
I'm located in the Toronto area.
Anyone interested for very nominal fee of my time for packaging and shipping :)

**Reply by Sagetvmaster** [reply]
Can you approximate shipping for a couple of HD300’s and a Hauppauge recorder (HDPVR Ver. 1?) to M4R-1B5?  Thanks

**Comment by TGsell1** [comment]
I’m interested in 2 of the HD300s if they’re still available.  Thanks Sent from my iPhoneOn Mar 14, 2024, at 8:12 PM, ZadkielOne ***@***.***> wrote:﻿
I have at least 5 HD300's, 1 HD200, 3 Hauppauge video recorders, probably some other misc stuff including the a few original boxes
I'm located in the Toronto area.
Anyone interested for very nominal fee of my time for packaging and shipping :)

**Comment by ZadkielOne** [comment]
Or a 26 minute drive? 😊
***@***.***

**Reply by Sagetvmaster** [reply]
Unfortunately it would have to be shipped I think.  Is that a problem?

**Reply by ZadkielOne** [reply]
I could meet you at Fairview Mall?

**Reply by Sagetvmaster** [reply]
I may be able to arrange a relative - I'm travelling at the moment, hence the shipping.  While I look for a relative, if you have time,  approximate M4R1B5 for me :)

**Reply by ZadkielOne** [reply]
OK, it's about $20 for 12"x12"x6" - 5lbs (should fit all three items not in their boxes, unless you want the original SageTV boxes?)
How's $20 per item sound - $60+$20shipping=$80

**Comment by ZadkielOne** [comment]
What’s your postal or zip code?

**Comment by TGsell1** [comment]
USA 11545.  I’ll pay you whatever needed via Venmo or  Zelle or check.  Thanks!Sent from my iPhoneOn Mar 14, 2024, at 10:13 PM, ZadkielOne ***@***.***> wrote:﻿
What’s your postal or zip code?



From: TGsell1 ***@***.***>

Sent: Thursday, March 14, 2024 9:53 PM

To: OpenSageTV/Community ***@***.***>

Cc: David Ward ***@***.***>; Comment ***@***.***>

Subject: Re: [OpenSageTV/Community] SOLD: 4x HD200 with remotes, 2x HD-PVR, 1x USB U-IRT (Discussion #118)



I’m interested in 2 of the HD300s if they’re still available.  Thanks Sent from my iPhoneOn Mar 14, 2024, at 8:12 PM, ZadkielOne ***@***.***> wrote:﻿

I have at least 5 HD300's, 1 HD200, 3 Hauppauge video recorders, probably some other misc stuff including the a few original boxes

I'm located in the Toronto area.

Anyone interested for very nominal fee of my time for packaging and shipping :)

**Comment by TGsell1** [comment]
Perfect!  How can I pay you and send you address.   My ***@***.*** Sent from my iPhoneOn Mar 14, 2024, at 11:24 PM, ZadkielOne ***@***.***> wrote:﻿
OK, it's about $20 for 12"x12"x6" - 5lbs (should fit all three items not in their boxes, unless you want the original SageTV boxes?)
How's $20 per item sound - $60+$20shipping=$80

**Comment by eelpout** [comment]
ZadkielOne: are all your HD300's sold? If not, I'd take one.

**Comment by Sagetvmaster** [comment]
Thanks anyway, I’m out on this one :)

**Comment by TGsell1** [comment]
ZadkielOne.  No need for original boxes! I’ll pay via whatever means work for you Sent from my iPhoneOn Mar 14, 2024, at 11:24 PM, ZadkielOne ***@***.***> wrote:﻿
OK, it's about $20 for 12"x12"x6" - 5lbs (should fit all three items not in their boxes, unless you want the original SageTV boxes?)
How's $20 per item sound - $60+$20shipping=$80

**Comment by TGsell1** [comment]
Hi – are we moving forward with this sale?  I am happy to do so and pay but didn’t see a response from you.  thanks! 
Tom

**Comment by ZadkielOne** [comment]
Sorry, just gathering up all the components.

You can contact me directly.

st at warddirect dot com
